subprojects/soak/src/integTest/groovy/org/gradle/vfs/FileSystemWatchingSoakTest.groovy

```diff
@@ -22,24 +22,21 @@ import org.gradle.integtests.fixtures.daemon.DaemonIntegrationSpec
 import org.gradle.internal.os.OperatingSystem
 import org.gradle.internal.watch.registry.FileWatcherRegistry
 import org.gradle.test.fixtures.file.TestFile
-import org.gradle.util.Requires
-import org.gradle.util.TestPrecondition
 import org.junit.Assert
-import spock.lang.Issue
 
-@Issue("https://github.com/gradle/gradle-private/issues/3235")
-@Requires(TestPrecondition.NOT_MAC_OS_X)
 class FileSystemWatchingSoakTest extends DaemonIntegrationSpec implements FileSystemWatchingFixture {
 
     private static final int NUMBER_OF_SUBPROJECTS = 50
     private static final int NUMBER_OF_SOURCES_PER_SUBPROJECT = 100
+    private static final int NUMBER_OF_FILES_IN_VFS = NUMBER_OF_SOURCES_PER_SUBPROJECT * NUMBER_OF_SUBPROJECTS * 2
```

Comment:
I guess this is something like `*.java` + `*.class`, right? I think some explanation here would be nice.